sqlglot/dialects/duckdb.py

```diff
@@ -85,6 +85,7 @@ class Tokenizer(tokens.Tokenizer):
             **tokens.Tokenizer.KEYWORDS,
             ":=": TokenType.EQ,
             "CHARACTER VARYING": TokenType.VARCHAR,
+            "GLOB": TokenType.GLOB,
```

Comment:
This is redundant because you've already added it in the base tokenizer's mapping